src/View/Components/BaseBadge.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace WireUi\View\Components;
+
+abstract class BaseBadge extends Component
+{
+    public string $pulseColor = 'bg-slate-500';
```

Comment:
I would use some default color here, like primary, secondary.

Reply:
Yes I agree, made the default on bg-secondary-500